src/main/java/com/oltpbenchmark/util/ResultUploader.java

```diff
@@ -67,16 +69,20 @@ public ResultUploader(Results r, XMLConfiguration conf, CommandLine argsLine) {
 
     }
 
-    public void writeDBParameters(PrintStream os) {
+    public void writeParams(PrintStream os) {
```

Comment:
AFAIK, nobody is using this feature. The website that handles the uploads hasn't been worked on in eight years. I think we can just delete this class.

Reply:
So as far as i can tell i already killed all the "upload" features of this class during my refactor.  as of now this class just collects additional data and ultimately writes to various files.  i've been looking at some of these files recently and think they are pretty useful.  again, i think all the "upload" stuff is long gone.

Reply:
Maybe rename it to `ResultWriter`?

Reply:
done.  also moved "write" methods out of `Results` into newly named `ResultsWriter`